Django_Blog/blog/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.urls import path
-from .views import IndexView,AuthorListView,AuthorDetailView,BlogListView,BlogDetailView,commentview
+from .views import IndexView,AuthorListView,AuthorDetailView,BlogListView,BlogDetailView,commentview,AuthorForm,BlogForm
 urlpatterns = [
     path('',IndexView.as_view(),name='index'),
     path('blogger/',AuthorListView.as_view(),name='authorlist'),
     path('blogger/<int:pk>/',AuthorDetailView.as_view(),name='authordetail'),
-    path('blogs/',BlogListView.as_view(),name='bloglist'),
+    path('blogger/form/',AuthorForm.as_view(),name='authorform'),
+    path('blogs/',BlogListView.as_view(),name='blogs'),
     path('blogs/<int:pk>/',BlogDetailView.as_view(),name='blogdetail'),
     path('blogs/<int:pk>/create/',commentview,name='comment'),
```

Comment:
Why this commentview is in small case. Change it to upper case.
Also please install the `black` formatter and format all the python files using it.